src/main/app/forms/validation/validators/IsTodayOrInFuture.ts

```diff
@@ -0,0 +1,43 @@
+import {
```

Comment:
please ensure that check-and-send page is updated for this story, I've added a comment in the code where it needs to be changed and resolve the conflicts on this branch

Reply:
has been done